x-pack/plugins/profiling/public/components/topn_functions/index.tsx

```diff
@@ -48,37 +48,44 @@ interface Row {
 function TotalSamplesStat({
   totalSamples,
   newSamples,
+  samplingRateA,
+  samplingRateB,
 }: {
   totalSamples: number;
   newSamples: number | undefined;
+  samplingRateA: number;
+  samplingRateB: number | undefined;
 }) {
   const value = totalSamples.toLocaleString();
 
-  const sampleHeader = i18n.translate('xpack.profiling.functionsView.totalSampleCountLabel', {
-    defaultMessage: ' Total sample estimate: ',
+  let sampleHeader = i18n.translate('xpack.profiling.functionsView.totalSampleCountLabel', {
```

Comment:
I think this looks a bit complicated:
WDYT of this:
```js
function getTotalSamplesLabel(samplingRate?: number) {
  if (samplingRate === undefined) {
    return i18n.translate('xpack.profiling.functionsView.totalSampleCountLabel', {
      defaultMessage: 'Total sample estimate:',
    });
  }
  return i18n.translate('xpack.profiling.functionsView.totalSampleCountLabelWithSamplingRate', {
    defaultMessage: 'Total sample (estimate sample rate: {samplingRate}):',
    values: { samplingRate },
  });
}

function TotalSamplesStat({
  totalSamples,
  newSamples,
  samplingRateA,
  samplingRateB,
}: {
  totalSamples: number;
  newSamples: number | undefined;
  samplingRateA: number;
  samplingRateB: number | undefined;
}) {
  const value = totalSamples.toLocaleString();

  if (newSamples === undefined || newSamples === 0) {
    return (
      <EuiStat
        title={<EuiText style={{ fontWeight: 'bold' }}>{value}</EuiText>}
        description={getTotalSamplesLabel(samplingRateA)}
      />
    );
  }

  const diffSamples = totalSamples - newSamples;
  const percentDelta = (diffSamples / (totalSamples - diffSamples)) * 100;

  return (
    <EuiStat
      title={
        <EuiText style={{ fontWeight: 'bold' }}>
          {value}
          <GetLabel value={percentDelta} prepend="(" append=")" />
        </EuiText>
      }
      description={getTotalSamplesLabel(samplingRateB)}
    />
  );
}
```

Reply:
Much better!